### DISCUSSION TITLE
Filter out an IP src / dst from pcap capture to save space

### DISCUSSION THREAD

**Discussion by Phillip-Samson** [Security-Onion-Solutions/securityonion#6030] [Unsupported Versions] [answered]
We would like to filter out our data domain replications from pcap capture, is this possible?

SO standalone 2.3

**Comment by dougburks** [comment, marked as answer]
Yes, please see the documentation:
https://docs.securityonion.net/en/2.3/bpf.html

**Comment by Phillip-Samson** [comment]
Thanks, Doug!

Regards,
Phillip Samson | I.T. Security Architect
GISF, GSEC, GCIH, GWAP

The only time success comes before work is in the dictionary!
